MyCryptoHQ/MyCrypto
# Use ShapeShift for all swaps

Closes https://github.com/MyCryptoHQ/MyCrypto/issues/1009

### What?
This PR includes minimal changes to enforce shapeshift as the swap provider for all swaps.

A subsequent PR can be issued to refactor swap and remove bity-specific logic.

### Testing notes
- Verify that shapeshift is used when selecting any pair
- Verify that bity requests are no longer being made (via network tab in dev tools)
